Makefile

```diff
@@ -16,16 +16,16 @@ binary-minimal: doc
 
 .PHONY: binary
 binary: doc
-	go build -tags extended -v -ldflags "-X  github.com/anuvu/zot/pkg/api.Commit=${COMMIT} -X github.com/anuvu/zot/pkg/api.BinaryType=extended" -o bin/zot ./cmd/zot
+	go build -tags extended,containers_image_openpgp -v -ldflags "-X  github.com/anuvu/zot/pkg/api.Commit=${COMMIT} -X github.com/anuvu/zot/pkg/api.BinaryType=extended" -o bin/zot ./cmd/zot
```

Comment:
why was this "containers_image_openpgp" needed?

Reply:
I had build errors on both conformance and travis CI:

```
# github.com/containers/image/v5/signature
/go/pkg/mod/github.com/containers/image/v5@v5.12.0/signature/mechanism_gpgme.go:16:16: undefined: gpgme.Context
```
Found this tag in an issue which was talking about same error.